resources/migrations/000_migrations.yaml

```diff
@@ -154,7 +161,8 @@ databaseChangeLog:
                 foreignKeyName: fk_userorgperm_ref_user_id
                 initiallyDeferred: false
                 nullable: false
-                references: core_user(id)
+                referencedTableName: core_user
+                referencedColumnNames: id
```

Comment:
`references: table(column)` doesn't work anymore -- see https://github.com/liquibase/liquibase/issues/1982 which was closed as a Won't Fix; so from now on we have to do FKs this way.